lib/include/openamp/remoteproc_virtio.h

```diff
@@ -20,6 +20,9 @@
 extern "C" {
 #endif
 
+/* maximum number of vring descriptors for a vdev */
+#define	RPROC_MAX_VRING_DESC	65535
```

Comment:
could USHRT_MAX be used instead ?  and please specify in comment that limitation is linked to 16-bit format

Reply:
@arnopo  Noted and addressed - please review - thanks!